### ISSUE TITLE
Can't run finac server

### ISSUE THREAD

**Issue by alterEgo123** [alttch/finac#1]
I'm trying to run a finac server.
I'm running a MySQL server locally and when I run this script

```
import finac as f
import finac.api as api

f.init('mysql+pymysql://finac:finac@localhost/finac')
api.key = 'secret'
application = api.app

if __name__ == '__main__':
    application.run(host='0.0.0.0', debug=True)

```

It returns the following error 

```

Traceback (most recent call last):
  File "server.py", line 1, in <module>
    import finac as f
  File "/usr/local/lib/python3.8/dist-packages/finac/__init__.py", line 14, in <module>
    from finac.core import init, config
  File "/usr/local/lib/python3.8/dist-packages/finac/core.py", line 496, in <module>
    class ForeignKeysListener(sa.interfaces.PoolListener):
AttributeError: module 'sqlalchemy' has no attribute 'interfaces'

```

Installed finac version: 0.5.6
Installed sqlalchemy version: 1.4.22

**Comment by divi255** [comment]
Hi. Until fixed, please use sqlalchemy==1.2.18 or other compatible

**Comment by divi255** [comment]
fixed in 0.5.7
